redwood-layout-uiview/src/commonMain/kotlin/app/cash/redwood/layout/uiview/UIViewFlexContainer.kt

```diff
@@ -110,7 +107,26 @@ internal class UIViewFlexContainer(
     value.setNeedsLayout()
   }
 
-  private inner class FlexContainerHostView : UIScrollView(cValue { CGRectZero }) {
+  public inner class FlexContainerHostView : UIScrollView(cValue { CGRectZero }) {
+
+    private var widthContraint: NSLayoutConstraint? = null
+    private var heightConstraint: NSLayoutConstraint? = null
+
+    override fun didMoveToSuperview() {
+      if (superview == null) return
```

Comment:
Is it safe not to call super in this case?

I would prefer if we switched to
```kotlin
if (superview != null) {
  // do stuff
}
super.didMoveToSuperview()
```
just to be safe